sdk/communication/azure-communication-rooms/src/main/java/com/azure/communication/rooms/RoomsClient.java

```diff
@@ -309,14 +309,14 @@ public Response<RemoveParticipantsResult> removeParticipantsWithResponse(String
             Objects.requireNonNull(roomId, "'roomId' cannot be null.");
             Map<String, ParticipantProperties> participantMap = convertRoomIdentifiersToMapForRemove(
                 identifiers);
-            ObjectMapper mapper = new ObjectMapper();
-            String updateRequest = mapper.writeValueAsString(new UpdateParticipantsRequest().setParticipants(participantMap));
+            String updateRequest = getUpdateRequest(participantMap);
+
             Response<Object> response = this.participantsClient
                 .updateWithResponse(roomId, updateRequest, context);
 
             return new SimpleResponse<RemoveParticipantsResult>(
                 response.getRequest(), response.getStatusCode(), response.getHeaders(), null);
-        } catch (JsonProcessingException ex) {
+        } catch (IOException ex) {
             ex.printStackTrace();
```

Comment:
same as in other file, if user wants it printed, they can do it themselves, we should not print stack traces